packages/data-explorer/src/views/MainView.vue

```diff
@@ -133,34 +143,31 @@ export default Vue.extend({
         this.deleteRow({ rowId: itemId })
       }
     },
-    async fetchViewData (tableName) {
-      if (this.tableName !== tableName) {
-        this.loading = true
-        await this.fetchTableMeta({ tableName })
-        if (this.isUserAuthenticated) {
-          this.fetchBreadcrumbs()
-        }
-        this.setTableName(tableName)
-      }
-      if (this.dataDisplayLayout === 'CardView') {
-        this.fetchCardViewData()
+    async paginateEntities () {
+      // The data-API uses zero-based page numbering, where pagination is one-based.
+      const pagination = { ...this.pagination, page: this.pagination.page - 1 }
+      const request = await this.fetchViewData({ tableName: this.$route.params.entity, pagination })
+      if (!request || !request.items.length) {
+        this.itemsCount = 0
       } else {
-        this.fetchTableViewData()
+        this.itemsCount = request.response.data.page.totalElements
```

Comment:
possible null pointer ( see preview , books example)

Reply:
Should be solved with the prior changes